docs/examples/actuators-cli.md

```diff
@@ -0,0 +1,94 @@
+# Actuators via ROS 2 Command Line
+
+This page describes how to use the the Create® 3 robot's actuators using the ROS 2 command line tools.
+
+## Audio
+
+### Play a sequence of notes
+
+Good times:
+```sh
+ros2 topic pub /cmd_audio irobot_create_msgs/msg/AudioNoteVector "{append: false, notes: [{frequency: 392, max_runtime: {sec: 0,nanosec: 177500000}}, {frequency: 523, max_runtime: {sec: 0,nanosec: 355000000}}, {frequency: 587, max_runtime: {sec: 0,nanosec: 177500000}}, {frequency: 784, max_runtime: {sec: 0,nanosec: 533000000}}]}" -1
+```
+
+Bad times:
+```sh
+ros2 topic pub /cmd_audio irobot_create_msgs/msg/AudioNoteVector "{append: false, notes: [{frequency: 369, max_runtime: {sec: 0,nanosec: 355000000}}, {frequency: 300, max_runtime: {sec: 0,nanosec: 533000000}}]}" -1
+```
+
+## Driving
+
+### Command velocities
+
+You can drive around the Create® 3 robot by publishing standard `Twist` messages on the `/cmd_vel` topic.
+
+```sh
+ros2 topic pub -r 20 /cmd_vel geometry_msgs/msg/Twist "{linear: {x: 0.2, y: 0.0, z: 0.0}, angular: {x: 0.0, y: 0.0, z: 0.0}}"
+```
+
+### Drive a distance
+The Drive Distance action takes a distance in meters and maximum speed in meters per second.
+```sh
+ros2 action send_goal /drive_distance irobot_create_msgs/action/DriveDistance "{distance: 0.5,max_translation_speed: 0.15}"
+```
+
+### Rotate an angle
+The Rotate Angle action takes a turn angle in radians and maximum angular speed in radians per second.
+```sh
+ros2 action send_goal /rotate_angle irobot_create_msgs/action/RotateAngle "{angle: 1.57,max_rotation_speed: 0.5}"
+```
```

Comment:
Should we also include the Drive Position and Drive Arc commands

Reply:
Sure.